app/helpers/lines_helper.rb

```diff
@@ -1,15 +1,13 @@
 module LinesHelper
   def lines
-    ENV['LINES'].present? ? split_and_strip(ENV['LINES']) : %w(DC MD VA)
+    LINES
   end
 
-  def current_line
+  def current_line_display
```

Comment:
Changing function names gives me the shivers. I see you got the tests all fixed up, so I'll grep through the code base a bit.